t/mojo/dynamic_methods.t

```diff
@@ -22,22 +22,24 @@ sub BUILD_DYNAMIC {
 }
 
 package main;
-
-# Basics
-my ($t1, $t2) = (Mojo::TestDynamic->new, Mojo::TestDynamic->new);
-Mojo::DynamicMethods::register 'Mojo::TestDynamic', $t1->hashref, 'foo', sub { };
-my $foo = \&Mojo::TestDynamic::_Dynamic::foo;
-my ($called_foo, $dyn_methods);
-Mojo::DynamicMethods::register 'Mojo::TestDynamic', $t1->hashref, 'foo', sub { $called_foo++; $dyn_methods = $_[1] };
-is $foo, \&Mojo::TestDynamic::_Dynamic::foo, 'foo not reinstalled';
-ok !Mojo::TestDynamic->can('foo'), 'dynamic method is hidden';
-ok eval { $t1->foo; 1 }, 'foo called ok';
-cmp_ok $called_foo, '==', 1, 'called dynamic method';
-ok !eval { $t2->foo; 1 }, 'error calling foo on wrong object';
-
-# Garbage collection
-undef($t1);
-undef($t2);
-ok(!keys(%$dyn_methods), 'dynamic methods expired');
+my ($t1, $t2, $dyn_methods);
```

Comment:
No, just no.

Reply:
What would be expected to be done in this case?